lang/gcc43/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup compiler_blacklist_versions 1.0
 
 name                gcc43
 version             4.3.6
-revision            13
+revision            15
```

Comment:
why increase revision by 2 ?

Reply:
This PR probably make more sense when viewed as five individual commits rather than one large one.
The first revision increase was for the dependency revision.
The second revision increase was because the latest version of isl has different library names.
The five commits are part of one PR so that GCC only has to be rebuilt once.

Reply:
The end result though should have the revisions only increase by one, otherwise an inconsistent gap is left in the master history. So please change this so they only increment by 1.

Reply:
I understand your argument. I just don't particularly like what will be the end result that users will see, which is the gcc ports jump by 2 revisions. Lets see what others think.

Reply:
Fair enough.
Thank you for taking the time to review the proposed changes.

Reply:
So that is now two of us that don't like this +2 revision bump.

@MarcusCalhoun-Lopez I think there are two options here. 1) squash all the commits and only increment the revisions by 1, or 2) split this PR in two. The dependency updates, and the update to isl, are two separate things and could (arguably should) be two separate PRs. I propose you remove the isl update from this PR, so make it only about the reworking of the dependencies. let that get reviewed and eventually merged, and then submit a second.

Reply:
I think it's fine to do them both at once and just bump by 1.